src/Resources/skeleton/serializer/Encoder.php.txt

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace App\Serializer;
+
+use Symfony\Component\Serializer\Encoder\DecoderInterface;
+use Symfony\Component\Serializer\Encoder\EncoderInterface;
+
+class {{ encoder_class_name }} implements EncoderInterface, DecoderInterface
+{
+    public function encode($data, $format, array $context = array())
+    {
+        // TODO: return your encoded data
+        return '';
+    }
+
+    public function supportsEncoding($format)
+    {
+        return '{{ format }}' === $format;
```

Comment:
The format could be in a `FORMAT` class const, as done in built-in encoders:

https://github.com/symfony/symfony/blob/8eb97f5c483b57d8cec2fe999d1437fadfeae575/src/Symfony/Component/Serializer/Encoder/JsonEncoder.php#L21

Reply:
@ogizanagi - makes sense. Fixed.